stl/inc/span

```diff
@@ -563,62 +569,69 @@ public:
+                _STL_VERIFY(_Count <= _Mysize - _Offset, "Count out of range in span::subspan()");
             }
         }
 #endif // _CONTAINER_DEBUG_LEVEL > 0
         using _ReturnType = span<element_type,
             _Count != dynamic_extent ? _Count : (_Extent != dynamic_extent ? _Extent - _Offset : dynamic_extent)>;
-        return _ReturnType{_Mydata + _Offset, _Count == dynamic_extent ? this->size() - _Offset : _Count};
+        return _ReturnType{_Mydata + _Offset, _Count == dynamic_extent ? _Mysize - _Offset : _Count};
     }
 
     _NODISCARD constexpr auto subspan(const size_type _Offset, const size_type _Count = dynamic_extent) const noexcept
     /* strengthened */ {
 #if _CONTAINER_DEBUG_LEVEL > 0
-        _STL_VERIFY(_Offset <= this->size(), "Offset out of range in span::subspan(offset, count)");
-        _STL_VERIFY(_Count == dynamic_extent || _Count <= this->size() - _Offset,
+        _STL_VERIFY(_Offset <= _Mysize, "Offset out of range in span::subspan(offset, count)");
+        _STL_VERIFY(_Count == dynamic_extent || _Count <= _Mysize - _Offset,
             "Count out of range in span::subspan(offset, count)");
 #endif // _CONTAINER_DEBUG_LEVEL > 0
         using _ReturnType = span<element_type, dynamic_extent>;
-        return _ReturnType{_Mydata + _Offset, _Count == dynamic_extent ? this->size() - _Offset : _Count};
+        return _ReturnType{_Mydata + _Offset, _Count == dynamic_extent ? _Mysize - _Offset : _Count};
     }
 
     // [span.obs] Observers
+    _NODISCARD constexpr size_t size() const noexcept {
+        return _Mysize;
+    }
+
     _NODISCARD constexpr size_type size_bytes() const noexcept {
 #if _CONTAINER_DEBUG_LEVEL > 0
-        _STL_VERIFY(this->size() <= dynamic_extent / sizeof(element_type),
+#pragma warning(suppress : 4127) // conditional expression is a constant
+        _STL_VERIFY(_Mysize <= dynamic_extent / sizeof(element_type),
             "size of span in bytes exceeds std::numeric_limits<size_t>::max()");
 #endif // _CONTAINER_DEBUG_LEVEL > 0
-        return this->size() * sizeof(element_type);
+        return _Mysize * sizeof(element_type);
     }
 
     _NODISCARD constexpr bool empty() const noexcept {
-        return this->size() == 0;
+        return _Mysize == 0;
     }
 
     // [span.elem] Element access
     _NODISCARD constexpr reference operator[](const size_type _Off) const noexcept /* strengthened */ {
 #if _CONTAINER_DEBUG_LEVEL > 0
-        _STL_VERIFY(_Off < this->size(), "span index out of range");
+        _STL_VERIFY(_Off < _Mysize, "span index out of range");
 #endif // _CONTAINER_DEBUG_LEVEL > 0
         return _Mydata[_Off];
     }
 
     _NODISCARD constexpr reference front() const noexcept /* strengthened */ {
 #if _CONTAINER_DEBUG_LEVEL > 0
-        _STL_VERIFY(this->size() > 0, "front of empty span");
+#pragma warning(suppress : 4127) // conditional expression is a constant

```

Comment:
The warning message says "is constant" without "a". Precisely copying the message can be important for searching. Occurs below.

Reply:
Nice find